landing_page/spec/models/micropost_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Micropost, type: :model do
+  let(:micropost) { Micropost.new }
+
+  context 'micropost with unacceptable data' do
+    it 'fails if no title is present' do
+      micropost.title = ''
+      expect(micropost).to_not be_valid
+    end
+
+    it 'fails if no content is present' do
+      micropost.content = ''
+    end
```

Comment:
no expect here so test isn't testing anything

Reply:
Resolved in commit 26a6f13cf0babdd05e0c1309613bcd7fa31c0b8a